### PR TITLE
Add Systemd to upgrade.sh and fix Systemd environment variables.

### PR DESCRIPTION
**Pull request by carrizo**

### CODE REVIEW

File: linux_files/upgrade.sh [hunk 1]
```diff
@@ -74,9 +75,17 @@ if [[ -z ${WSL2} ]]; then
...
     sudo check-dnf
   fi
 fi
 
+# Upgrade Systemd
+sudo curl -L -f "${base_url}/linux_files/start-systemd.sudoers" -o /etc/sudoers.d/start-systemd
+sudo curl -L -f "${base_url}/linux_files/start-systemd.sh" -o /usr/local/bin/start-systemd
+sudo curl -L -f "${base_url}/linux_files/systemctl3.py" -o /usr/local/bin/wslsystemctl
+sudo chmod u+x /usr/local/bin/start-systemd
+sudo chmod u+x /usr/local/bin/wslsystemctl
```
Comment (crramirez) [inline linux_files/upgrade.sh]: Make it executable for every user so bash-completion will show it with sudo
```suggestion
sudo chmod +x /usr/local/bin/wslsystemctl
```

File: linux_files/00-remix.sh [hunk 2]
```diff
@@ -85,3 +85,15 @@ if [ -z "$WIN_HOME" ] && (command -v cmd.exe >/dev/null 2>&1); then
   unset win_home_lnk
 
 fi
+
+# Fix $PATH for Systemd
+if [ ! -f "$HOME/.systemd.env" ]
```
Comment (crramirez) [inline linux_files/00-remix.sh]: Formatting
```suggestion
if [ ! -f "$HOME/.systemd.env" ]; then
```

File: linux_files/00-remix.sh [hunk 3]
```diff
@@ -85,3 +85,15 @@ if [ -z "$WIN_HOME" ] && (command -v cmd.exe >/dev/null 2>&1); then
...
+then
```
Comment (crramirez) [inline linux_files/00-remix.sh]:
```suggestion
```

File: linux_files/00-remix.sh [hunk 4]
```diff
@@ -85,3 +85,15 @@ if [ -z "$WIN_HOME" ] && (command -v cmd.exe >/dev/null 2>&1); then
...
+    echo "PATH='$PATH'" > "$HOME/.systemd.env"
```
Comment (crramirez) [inline linux_files/00-remix.sh]:
```suggestion
  echo "PATH='$PATH'" > "$HOME/.systemd.env"
```

File: create-targz.sh [hunk 5]
```diff
@@ -58,7 +58,7 @@ function build() {
   cp "${ORIGINDIR}"/linux_files/local.conf "${TMPDIR}"/dist/etc/fonts/
   cp "${ORIGINDIR}"/linux_files/00-remix.sh "${TMPDIR}"/dist/etc/profile.d/
   cp "${ORIGINDIR}"/linux_files/00-remix.fish "${TMPDIR}"/dist/etc/fish/conf.d/
-  chmod -x "${TMPDIR}"/dist/etc/profile.d/00-remix.sh
+  chmod -x,+r "${TMPDIR}"/dist/etc/profile.d/00-remix.sh
   chmod -x "${TMPDIR}"/dist/etc/fish/conf.d/00-remix.fish
```
Comment (crramirez) [inline create-targz.sh]: Probably it should be needed also for fish
```suggestion
  chmod -x,+r "${TMPDIR}"/dist/etc/fish/conf.d/00-remix.fish
```

File: create-targz.sh [hunk 6]
```diff
@@ -128,6 +128,7 @@ echo '%wheel   ALL=NOPASSWD: /usr/bin/check-dnf' | sudo EDITOR='tee -a' visudo -
 chmod -w /usr/bin/check-dnf
 chmod u+x /usr/bin/check-dnf
 chmod -x /etc/fish/conf.d/check-dnf.fish
```
Comment (crramirez) [inline create-targz.sh]:
```suggestion
chmod -x,+r /etc/fish/conf.d/check-dnf.fish
```

File: linux_files/00-remix.sh [hunk 7]
```diff
@@ -85,3 +85,15 @@ if [ -z "$WIN_HOME" ] && (command -v cmd.exe >/dev/null 2>&1); then
...
+else
```
Comment (crramirez) [inline linux_files/00-remix.sh]: You can use `elif`

**Review by crramirez — CHANGES_REQUESTED** [review]

File: linux_files/00-remix.sh (hunk 7 above)
Comment (carrizo) [reply linux_files/00-remix.sh]: fixed

**Review by crramirez — APPROVED** [review]

**Pull request merged**